salon_creator/lib/screens/profile_screen.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+import 'package:salon_creator/common/color.dart';
+import 'package:salon_creator/widgets/custom_button.dart';
+import 'package:salon_creator/widgets/custom_label.dart';
+
+class ProfileScreen extends StatefulWidget {
+  const ProfileScreen({Key key}) : super(key: key);
+
+  @override
+  _ProfileScreenState createState() => _ProfileScreenState();
+}
+
+class _ProfileScreenState extends State<ProfileScreen> {
+  TextEditingController descriptionController = new TextEditingController();
```

Comment:
```suggestion
  TextEditingController descriptionController = TextEditingController();
```
You can omit using the **new** keyword in dart